acceptance-tests/src/acceptance-test/java/tech/pegasys/teku/test/acceptance/CapellaRemoteSignerAcceptanceTest.java

```diff
@@ -39,7 +39,7 @@ public class CapellaRemoteSignerAcceptanceTest extends AcceptanceTestBase {
   private static final URL JWT_FILE = Resources.getResource("auth/ee-jwt-secret.hex");
 
   @Test
-  void denebWithRemoteSigner() throws Exception {
+  void capellaWithRemoteSigner() throws Exception {
     final UInt64 currentTime = new SystemTimeProvider().getTimeInSeconds();
     final int genesisTime =
         currentTime.intValue() + 10; // genesis in 10 seconds to give node time to start
```

Comment:
Do we want 10 or 30 seconds? :)

Reply:
i did this one before the rest, i think we decided 30...